### DISCUSSION TITLE
deleted child_parent_sku_relationship is problem

### DISCUSSION THREAD

**Discussion by zhong-mr** [amzn/selling-partner-api-models#4291] [Q&A]
I wanted to delete the parent_sku of the child_parent_sku_relationship attribute, and found that after the submission was successful, the result of getListingItem still retained the parent_sku of child_parent_sku_relationship

json_listing_feed:
{
    "header": {
        "sellerId": "xxx",
        "version": "2.0",
        "issueLocale": "en_US"
    },
    "messages": [
        {
            "messageId": 1,
            "sku": "xxxabc",
            "operationType": "PATCH",
            "productType": "FILE_FOLDER",
            "patches": [
                {
                    "op": "delete",
                    "path": "/attributes/parentage_level",
                    "value": [
                        {
                            "value": "child",
                            "marketplace_id": "ATVPDKIKX0DER"
                        }
                    ]
                },
                {
                    "op": "delete",
                    "path": "/attributes/child_parent_sku_relationship",
                    "value": [
                        {
                            "child_relationship_type": "variation",
                            "parent_sku": "xxxABC",
                            "marketplace_id": "ATVPDKIKX0DER"
                        }
                    ]
                }               
            ]
        }
    ]
}

**Comment by weilinggu** [comment]
Hi @zhong-mr 

_To un-parent a listing item, you can delete either the parent or child listing to eliminate the relationship between them. When you delete a parent listing, it removes the relationship between that parent and all its child listings. When you delete a child listing, it eliminates the relationship between just that one child listing and its parent._

Hope this helps
Thanks
Weiling
Amazon Selling Partner Services

**Reply by zhong-mr** [reply]
However, if it is removed to the case of the parent asin, it will cause my parent asin to be unsold. I'm just trying to get one of them asin out of there. You don't want to delete the entire asin. To get a clear answer, can JSON_LISTING_FEED only unbind the relationship between the child ASIN or the parent ASIN by deleting them?
